### ISSUE TITLE
Fix warnings from google play audits

### ISSUE THREAD

**Issue by GeorgeBellTMH** [themeetinghouse/mobile#91]
Google play has identified a number of warnings that should be fixed...most of them are related to accessibility and easy to fix. 

George/Lucas can provide the report to whoever wants to work on this.

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=588046809)

**Comment by lucastbelem** [comment]
Will any of this hinder launch?

**Comment by lucastbelem** [comment]
@GeorgeBellTMH I was trying to find this again and can't remember where it is. Could you pls point me to where we find this info?

**Comment by lucastbelem** [comment]
Closing this. Will open individual issues for it.
